README.md

```diff
@@ -13,6 +13,8 @@ Receive feedback in-line with your code, such as how many vulnerabilities a pack
 
 ![Vuln Cost animation](https://raw.githubusercontent.com/snyk/vulncost/master/images/vulncost.gif)
 
+🔒 Your code or manifest files never leave your machine. Snyk only needs the depencency name and version to test against our constantly updated vulnerability database.
```

Comment:
s/depencency/dependency/.  Also s/Your code or/Your code and/

Otherwise, LGTM.